examples/chart/README.md

```diff
@@ -0,0 +1 @@
+When using a Teleport provided values.yaml file from one of the above Helm Charts with a Helm Chart from the Teleport Helm Repo, please ensure you are on the most recent major release branch. (e.g. branch/v8). Using a values.yaml from the master branch may cause compatability issues. 
```

Comment:
Since this PR adds a README for the entire `chart` directory, I think it would be helpful to add a more general introduction to the directory. Here is a bare-bones example:

```markdown
## Teleport Helm charts
This is the source for Teleport's Helm charts.

### Which chart should I use?
Please see the [Helm chart reference](https://goteleport.com/docs/kubernetes-access/helm/reference/) for information about individual charts.

### Values file warning
When using a `values.yaml` file provided by Teleport, you must use the file from the most recent major release branch (e.g., `branch/v8`). Using the `values.yaml` file from the `master` branch may cause compatibility issues.
```
I think it would also be great to include a short description of each chart we manage here, though that might be out of scope for this PR.

Reply:
I agree, this would be nice. However, I do think this is out of scope for this PR, as you mentioned.